src/analysis/ExpressionExplorer.jl

```diff
@@ -1132,7 +1132,7 @@ function compute_symbolreferences(ex::Any)::SymbolsState
     symstate
 end
 
-function try_compute_symbolreferences(ex::Any)::SymbolsState
```

Comment:
This was `Any` because it can take the output of `Meta.parse(::String)` as an input, which confusingly is not always an `Expr`:

```julia
Meta.parse("123") isa Int64
```

In internal Pluto use, we always have expressions wrapped in a `:toplevel` Expr and more, which means that it is always an `Expr`. So `Any` is there to support use cases outside of Pluto's use, like people using Pluto.jl to do syntax analysis

Reply:
Hmm okay I should have let you review this one. Apologies!